packages/common/client-utils/api-report/client-utils.legacy.alpha.api.md

```diff
@@ -9,8 +9,6 @@ export { EventEmitter }
 // @alpha
 export type EventEmitterEventType = string;
 
-export { performance_2 as performance }
```

Comment:
This is an `@internal` export, right? So this change is effectively a correction of [this API-Extractor bug](https://github.com/microsoft/rushstack/issues/4762)?